### ISSUE TITLE
Issue with & in img src URLs

### ISSUE THREAD

**Issue by Capncavedan** [validator/validator#186]
Have an image src URL with & characters in it being flagged as non-encoded; I think they should be acceptable.

Error message from validator:
`Error: & did not start a character reference. (& probably should have been escaped as &amp;.)`

HTML img tag, as generated by the Ruby [gravatar_image_tag](https://rubygems.org/gems/gravatar_image_tag) gem:
````html
<img alt="Merchants Developer" src="https://secure.gravatar.com/avatar/22a18e930e8a721f1d0024e6acbd35a4?default=http%3A%2F%2Flocalhost%3A3001%2Fassets%2Fmerchants_quill_square.png&secure=true&size=48" width="48" height="48" /></a>
````

**Comment by cvrebert** [comment]
Duplicate of #31.

**Comment by Capncavedan** [comment]
Thanks for the info.  I’ll close.

**Comment by sideshowbarker** [comment]
Fixed in v16.1.1 and after https://github.com/validator/validator/releases/latest and pushed to https://validator.w3.org/nu/
